apps/corona-banner/src/index.less

```diff
@@ -0,0 +1,151 @@
+@font-face {
```

Comment:
We could move this file under the `less/` directory in the root to be consistent with other apps in the repo (e.g. mitt-konto and app-article). The naming convetion is `<app-name>.less`.